packages/http-client-java/generator/http-client-generator-test/src/main/java/com/specialheaders/repeatability/models/ImmediateSuccessResponseRepeatabilityResult.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com.specialheaders.repeatability.models;
+
+/**
+ * Defines values for ImmediateSuccessResponseRepeatabilityResult.
+ */
+public enum ImmediateSuccessResponseRepeatabilityResult {
+    /**
+     * Enum value accepted.
+     */
+    ACCEPTED("accepted"),
```

Comment:
Caused by [case1](https://gist.github.com/haolingdong-msft/df3f25a048baf83d8fabf5206aacc7db#1-responses-header-schema-has-public-access-in-tcgc)